core/src/main/scala/akka/persistence/jdbc/db/Postgres11Profile.scala

```diff
@@ -0,0 +1,36 @@
+package akka.persistence.jdbc.db
+
+import com.github.tminglei.slickpg._
+import slick.basic.Capability
+import slick.jdbc.JdbcCapabilities
+
+trait Postgres11Profile
```

Comment:
Is it important that it is Postgres11 ? Doesnt it work for Postgres 12?

Reply:
I wanted to avoid naming collisions with the PostgresProfile provided by slick. I'll try to find better name 👍